### DISCUSSION TITLE
HTTP request filitering with '=in.'

### DISCUSSION THREAD

**Discussion by favoriteother** [supabase/supabase#20690] [Questions]
Im trying to make a get request filtering for values matching any from a provided list like so: 
```
// fetch all supabase posts with any of these frame IDs ahead of loop
    const frameIds = allFrameNodes.map(node => node.id);
    const frameIdsQueryString = frameIds.join(',');
    const postsResponse = await fetch(
        `https://vwmesxkarcfiqijhxzlq.supabase.co/rest/v1/Post?figma_frame_id=in.${frameIdsQueryString}`,
        {
            method: "GET",
            headers: {
                apikey: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InZ3bWVzeGthcmNmaXFpamh4emxxIiwicm9sZSI6ImFub24iLCJpYXQiOjE2OTgxNjM4MzIsImV4cCI6MjAxMzczOTgzMn0.ZSbsuyGKyXwmZAuMRpYRIYEmStIvnwHSjlXz-nxsO-o",
                    Authorization:
                        "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InZ3bWVzeGthcmNmaXFpamh4emxxIiwicm9sZSI6ImFub24iLCJpYXQiOjE2OTgxNjM4MzIsImV4cCI6MjAxMzczOTgzMn0.ZSbsuyGKyXwmZAuMRpYRIYEmStIvnwHSjlXz-nxsO-o",
                    "Content-Type": "application/json",
                    Prefer: "return=representation",
                },
            }
        );
```
However, THIS is throwing back a CORS policy error saying:
'Access to fetch at 'https://vwmesxkarcfiqijhxzlq.supabase.co/rest/v1/Post?figma_frame_id=in.[list of ids] from origin 'null' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

It is weird because this request worked unblocked by CORS when I was using '=eq.' filter and calling it once per frameID. 

Any thoughts? Is my syntax wrong and throwing CORS error for some reason?

**Comment by GaryAustin1** [comment]
Not sure on the cors, but it could be confused.
Your in string should look like:
![image](https://github.com/supabase/supabase/assets/54564956/be81a664-04eb-47fc-9f58-2ce8e8cf4224)
https://postgrest.org/en/stable/references/api/tables_views.html
